app/ssr/page.js

```diff
@@ -0,0 +1,33 @@
+import { experimental_use as use } from 'react'
+
+import Page from '../../components/page.client'
+import Story from '../../components/story.client'
```

Comment:
Can we move all of the components into the `app` directory to colocate?

Reply:
What is the point of moving all of the components into the `app` directory ?

https://github.com/vercel/next.js/tree/canary/examples/layout-component

Reply:
We don't have to but it's nice to highlight you can colocate things now.